### PR TITLE
[fix](bloom filter)Fix drop column with bloom filter index

### PR DESCRIPTION
**Pull request by qidaye**
### What problem does this PR solve?

1. When drop column with bloom filter, we modify the bloom filter column info
2. When replay editLog, we rebuild bloom filter info by table schema.

Related PR: #41369

Problem Summary:

### Release note

Fix drop column with bloom filter

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [x] Confirm the release note
- [x] Confirm test cases
- [ ] Confirm document
- [x] Add branch pick label

### CODE REVIEW

**Comment by qidaye** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java [hunk 1]
```diff
@@ -440,7 +441,9 @@ private boolean processDropColumn(DropColumnClause alterClause, OlapTable olapTa
                         newBfCols.add(bfCol);
                     }
                 }
-                olapTable.setBloomFilterInfo(newBfCols, olapTable.getBfFpp());
```
Comment (airborne12) [inline fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java:443]: why remove?

Comment (qidaye) [reply fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java:443]: Record the new bf columns to `propertyMap`, the actual bf columns drop operation is happening in the following schema change job which created by directly schema change.

**Comment by qidaye** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by airborne12 — APPROVED** [review]
LGTM

**Pull request merged**